README.md

```diff
@@ -35,7 +35,7 @@ Next, setup the replica set. Once the containers have started up exec into one o
 Inside the shell enter the following:
 
 ```sh
-config = {
+config1 = {
```

Comment:
A Q, why config1 and not just config ?

Reply:
addressed in https://github.com/jembi/openhim-mediator-mapping/pull/205/commits/dd30cd98f267f8a46b99a1cb686be24a5a90613c